doc/release/platforms/infiniband.rst

```diff
@@ -0,0 +1,178 @@
+.. _readme-infiniband:
+
+============================
+Using Chapel with InfiniBand
+============================
+
+Avoiding Slow Job Launch
+++++++++++++++++++++++++
+
+We've observed job launch hangs in some systems with InfiniBand
```

Comment:
For Tony, it takes ~ 1 minute. For other clusters, I've seen it be worse. I believe it will complete eventually, it's just users might give up...